plenum/common/roles.py

```diff
@@ -0,0 +1,11 @@
+from enum import Enum
+
+
+class Roles(Enum):
+    TRUSTEE = "0"
+    TGB = "1"
```

Comment:
I'm not sure about the TGB being a separate role. The TGB has meaningful work to do, but I believe the sovrin ecosystem doesn't require it. I think the TGB is acting like a TRUSTEE in the near term.